databricks/koalas/groupby.py

```diff
@@ -602,6 +602,79 @@ def rename_output(pdf):
             sdf=sdf, data_columns=return_schema.fieldNames(), index_map=[])  # index is lost.
         return DataFrame(internal)
 
+    def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+        """
+        Compute group sizes.
+
+        See Also
+        --------
+        databricks.koalas.Series.groupby
+        databricks.koalas.DataFrame.groupby
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'A': [1, 2, 2, 3, 3, 3],
+        ...                    'B': [1, 1, 2, 3, 3, 3]},
+        ...                   columns=['A', 'B'])
+        >>> df
+           A  B
+        0  1  1
+        1  2  1
+        2  2  2
+        3  3  3
+        4  3  3
+        5  3  3
+
+        >>> df.groupby('A')['B'].value_counts().sort_index()  # doctest: +NORMALIZE_WHITESPACE
+        A  B
+        1  1    1
+        2  1    1
+           2    1
+        3  3    3
+        Name: B, dtype: int64
+
+        >>> df.groupby('A')['B'].value_counts(normalize=True).sort_index()
+        ...                                # doctest: +NORMALIZE_WHITESPACE
+        A  B
+        1  1    1.0
+        2  1    0.5
+           2    0.5
+        3  3    1.0
+        Name: B, dtype: float64
+        """
+        if bins is not None:
+            raise NotImplementedError("value_counts currently does not support bins")
+
+        groupkeys = self._groupkeys + self._agg_columns
+        groupkey_cols = [s._scol.alias('__index_level_{}__'.format(i))
+                         for i, s in enumerate(groupkeys)]
+        sdf = self._kdf._sdf
+        agg_column = self._agg_columns[0].name
+        sdf = sdf.groupby(*groupkey_cols).count().withColumnRenamed('count', agg_column)\
+            .sort(*groupkey_cols)
+
+        if sort:
+            if ascending:
+                sdf = sdf.orderBy(F.col(agg_column))
+            else:
+                sdf = sdf.orderBy(F.col(agg_column).desc())
+
+        if normalize:
+            nor_groupkey_cols = [s._scol.alias('__index_level_{}__'.format(i))
+                                 for i, s in enumerate(self._groupkeys)]
+            nor_groupkey_names = ['__index_level_{}__'.format(i)
+                                  for i, s in enumerate(self._groupkeys)]
+            sum = self._kdf._sdf.groupby(nor_groupkey_cols).count()
+            sdf = sdf.join(sum, on=nor_groupkey_names, how='left')\
```

Comment:
Can we avoid `join`? If not, let's don't add normalize option for now. It's kind of expensive ...